src/main/java/ir/sahab/cleanup/Cleanups.java

```diff
@@ -63,27 +69,31 @@ public void doAllQuietly() {
         }
     }
 
+    /**
+     * Does all cleanup operations and if there is an exception on any operation, throws the first
+     * exception after trying for all operations. Probably the first error was the cause of the
```

Comment:
Replace:
Probally ... > We choose the first exception to throw because it is the most important ones and may be the reason for the next exceptions.